plugins/catalog/src/js/repositories/components/RepositoriesDeleteConfirm.js

```diff
@@ -37,7 +37,9 @@ const RepositoriesDeleteConfirm = ({
       <Trans render="span">Delete Repository</Trans>
     </ModalHeading>
   );
-  const rightButtonText = i18nMark("Delete Repository");
+  const rightButtonText = pendingRequest
+    ? i18nMark("Deleting...")
+    : i18nMark("Delete Repository");
```

Comment:
I didn't detect a change with this button, either.

Reply:
I can't get it to stay on screen long enough to show a change

Reply:
I console.log-ed pendingRequest here and it is false even when deleting.
![Peek 2019-06-21 09-48](https://user-images.githubusercontent.com/40791275/59903482-c60f0380-9409-11e9-9ce8-7cc0b1a4be6d.gif)

Reply:
Now it works.